.env.example

```diff
@@ -10,8 +10,18 @@ DB_DATABASE=PLEASESETME
 DB_USERNAME=PLEASESETME
 DB_PASSWORD=PLEASESETME
 
+DB_TEST_HOST=localhost
+DB_TEST_DATABASE=PLEASESETME
+DB_TEST_USERNAME=PLEASESETME
+DB_TEST_PASSWORD=PLEASESETME
+
 CACHE_DRIVER=file
 SESSION_DRIVER=file
 QUEUE_DRIVER=sync
 LOCALE=en
 LOCALE_FALLBACK=en
+
+ENABLE_REGISTER=false
+ENABLE_REGISTER_MAIL=false
+
+ENABLE_ADMIN_MAIL=false
```

Comment:
I think there is a missing MAIL_USERNAME= setting required. Let's go with this for now, but I think a different approach might be to login to an smtp server (like google mail) and authenticate via their system. Doing it this way might not always guarantee email delivery as we're sending the mail from the server its installed on which might get blacklisted.

This is how unRaid's email notification system is setup:
![image](https://cloud.githubusercontent.com/assets/12205479/20645106/4a21cf02-b4b7-11e6-8a79-165904c6a9d5.png)

Might be how we do it in future :smile: 

For now I'm getting this error when trying to register:
`Swift_TransportException in AuthHandler.php line 181:
Failed to authenticate on SMTP server with username "rudi.olckers@gmail.com" using 2 possible authenticators`